### ISSUE TITLE
Is there is way to add ssh key as git config?

### ISSUE THREAD

**Issue by balu-ce** [upbound/provider-terraform#50]
### What environment did it happen in?

* Universal Crossplane Version:
* Provider Version:

**Comment by bobh66** [comment]
@balu-ce We had a proposed change to support this in the crossplane-contrib repo - https://github.com/crossplane-contrib/provider-terraform/pull/77 - but it will need some rework to be brought up to date with the latest version of the provider.

Is this what you are looking for?

**Comment by balu-ce** [comment]
@bobh66  yeah !! . This is what we looking for . can we expect it in the next release

**Comment by ytsarev** [comment]
@balu-ce thanks for the report, I created https://github.com/upbound/provider-terraform/issues/53 to track it. Will close this one to avoid dups; please provide any feedback in #53
